app/src/main/res/values-bn-rBD/strings.xml

```diff
@@ -19,37 +19,50 @@
  -->
 
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="vibrate_on_keypress">"কীপ্রেস এর সময় কম্পন"</string>
-    <string name="sound_on_keypress">"কীপ্রেস এর সময়ের শব্দ"</string>
-    <string name="popup_on_keypress">"কীপ্রেস এর ফলে পপআপ"</string>
-    <string name="settings_screen_preferences">"পছন্দসমূহ"</string>
-    <string name="settings_screen_theme">"থিম"</string>
-    <string name="abbreviation_unit_milliseconds">"<xliff:g id="MILLISECONDS">%s</xliff:g>মিলি.সে."</string>
-    <string name="settings_system_default">"সিস্টেম ডিফল্ট"</string>
-    <string name="auto_cap">"স্বতঃ-ক্যাপিটালাইজেশন"</string>
-    <string name="auto_cap_summary">"প্রতিটি বাক্যের প্রথম শব্দ বড় হাতের করুন"</string>
-    <string name="select_language">"ভাষাগুলি"</string>
-    <string name="locale_name_en_GB">"ইংরেজি (UK)"</string>
-    <string name="locale_name_en_US">"ইংরেজি (US)"</string>
-    <string name="locale_name_es_US">"স্প্যানিশ (US)"</string>
-    <string name="locale_name_hi_ZZ">"Hinglish"</string>
-    <string name="locale_name_sr_ZZ">"সার্বিয়ান (ল্যাটিন)"</string>
-    <string name="subtype_traditional">"ঐতিহ্যবাহি"</string>
-    <string name="subtype_compact">"কম্প্যাক্ট"</string>
-    <string name="add">"জুড়ুন"</string>
-    <string name="remove">"সরান"</string>
-    <string name="prefs_keypress_vibration_duration_settings">"কীপ্রেস কম্পন সময়কাল"</string>
-    <string name="prefs_keypress_sound_volume_settings">"কীপ্রেস সাউন্ড ভলিউম"</string>
-    <string name="prefs_key_longpress_timeout_settings">"কী প্রেসে দীর্ঘ বিলম্ব"</string>
-    <string name="button_default">"ডিফল্ট"</string>
-    <string name="prefs_keyboard_height_settings">কীবোর্ডের উচ্চতা</string>
-    <string name="keyboard_color">কীবোর্ডের ভিন্ন রং নির্ধারণ করো</string>
-    <string name="hide_special_chars">বিশেষ চিহ্ন লুকাও</string>
-    <string name="hide_language_switch_key">ভাষা পরিবর্তনের কী লুকাও</string>
-    <string name="show_number_row">আলাদা নম্বরের সারি দেখাও</string>
-    <string name="space_swipe">স্পেস সোয়াইপ দিয়ে কার্সর সরাও</string>
-    <string name="delete_swipe">সোয়াইপ মুছো</string>
-    <string name="matching_navbar_color">ন্যাভিগেশন বারে মানানসই রং ব্যবহার করো</string>
-    <string name="setup_message">সিম্পল কিবোর্ড সক্রিয় করা নেই। ভাষা সেটিংস &amp; পরিবর্তন করতে OK/আচ্ছা টিপ দাও। তোমার কীবোর্ডে সিম্পল কীবোর্ড বা Simple Keyboard টিপ দিতে হবে।</string>
```

Comment:
Do not translate application names. It is still shown as "Simple Keyboard" in launcher.

Reply:
It's not translation. I transliterated the word _Simple Keyboard_ with Bengali letters. Is it okay?

Reply:
Does the launcher and system settings also "transliterate"?

Reply:
No, it's in English letters.

Reply:
Then leave it in English letters.